src/Core/Services/Implementations/HandlebarsMailService.cs

```diff
@@ -200,21 +203,24 @@ public async Task SendOrganizationConfirmedEmailAsync(string organizationName, s
         await _mailDeliveryService.SendEmailAsync(message);
     }
 
-    public Task SendOrganizationInviteEmailAsync(string organizationName, OrganizationUser orgUser, ExpiringToken token) =>
-        BulkSendOrganizationInviteEmailAsync(organizationName, new[] { (orgUser, token) });
+    public Task SendOrganizationInviteEmailAsync(string organizationName, OrganizationUser orgUser, ExpiringToken token, bool isFreeOrg) =>
+        BulkSendOrganizationInviteEmailAsync(organizationName, new[] { (orgUser, token) }, isFreeOrg);
 
-    public async Task BulkSendOrganizationInviteEmailAsync(string organizationName, IEnumerable<(OrganizationUser orgUser, ExpiringToken token)> invites)
+    public async Task BulkSendOrganizationInviteEmailAsync(string organizationName, IEnumerable<(OrganizationUser orgUser, ExpiringToken token)> invites, bool isFreeOrg)
     {
         MailQueueMessage CreateMessage(string email, object model)
         {
             var message = CreateDefaultMessage($"Join {organizationName}", email);
             return new MailQueueMessage(message, "OrganizationUserInvited", model);
         }
-
+        var freeOrgTitle = CoreHelpers.SanitizeForEmail("A Bitwarden member invited you to an organization. Join now to start securing your passwords!", false);
```

Comment:
Question: Does this string need to be sanitized?

Reply:
No it does not, i added it when i was testing another thing and forgot to remove it, thanks!